src/redux/features/header/headerSlice.js

```diff
@@ -1,4 +1,4 @@
-import { createSlice } from "@reduxjs/toolkit";
+import { createSlice, current } from "@reduxjs/toolkit";
 
 const initialState = {
```

Comment:
why do we need to store the state of both buttons?

Reply:
Suggest: use 1 variable to store the current view mode is landlord or tenant. And using that variable to active the corresponding button. Example:
 `
const initialState = {
         viewMode: "landlord"
}
`

Reply:
Changed the headerSlice into Config slice and created a new state to keep track the button